tests/foreman/cli/test_remoteexecution.py

```diff
@@ -482,6 +483,50 @@ def test_positive_run_scheduled_job_template_by_ip(self, fixture_vmsetup, fixtur
             )
             raise AssertionError(result)
 
+    @tier3
+    def test_positive_run_receptor_installer(self):
+        """Run Receptor installer ("Configure Cloud Connector")
+
+        :id: 811c7747-bec6-1a2d-8e5c-b5045d3fbc0d
+
+        :expectedresults: The job passes, installs Receptor that peers with c.r.c
+        """
+        template_name = 'Configure Cloud Connector'
+        invocation = make_job_invocation(
+            {
+                'async': True,
+                'job-template': template_name,
+                'inputs': 'satellite_user="{0}",satellite_password="{1}"'.format(
+                    settings.server.admin_username, settings.server.admin_password
+                ),
+                'search-query': "name ~ {0}".format(settings.server.hostname),
+            }
+        )
+        invocation_id = invocation['id']
+
+        try:
+            wait_for(
+                lambda: entities.JobInvocation(id=invocation_id).read().status == 0,
+                timeout="1500s",
```

Comment:
1. Sure, I can change that. It also solves your previous comment because now it makes sense to explicitly check for success.
2. I prefer taking this to fail a little longer. It prevents waste of human time by preventing false positives due to timeout.

Reply:
Agreed, if you use different condition in `wait_for`, this assertion should be left intact, as we want to ensure job actually succeeded.

I first noticed you use the same condition, then run this test locally and saw that it still waits, while Satellite already knows job failed.